experiments/68x50/slotnick_etal_2001/slotnick_interaction_analysis.m

```diff
@@ -0,0 +1,12 @@
+function [ms,ss] = slotnick_interaction_analysis(tst_cate, tst_coor)
+
```

Comment:
Adding a comment referencing the figure in the paper would be helpful. Yes, I'm holding newer (i.e. your) code to higher standards than older (i.e. my) code ... :smiley: